pkg/controller/configmap_ca_injector/controller.go

```diff
@@ -186,11 +187,18 @@ func (r *ReconcileConfigMapInjector) Reconcile(ctx context.Context, request reco
 				ObjectMeta: metav1.ObjectMeta{
 					Namespace: configMap.Namespace,
 					Name:      configMap.Name,
+					Annotations: map[string]string{
+						names.OpenShiftComponent: configMap.Annotations[names.OpenShiftComponent],
```

Comment:
This line is assigning annotation "openshift.io/owning-component" to the value of the  "openshift.io/owning-component"  annotation in the original configmap.